packages/a11y/package.json

```diff
@@ -20,6 +20,9 @@
 	"bugs": {
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
+	"engines": {
+		"node": "^12.13.0 || ^14.15.0 || >=15.0.0"
```

Comment:
What's the reason for `^12.13.0` instead of `12.*` ?
Also I _think_ you don't need the `||`, this field could be represented as `12.* 14.* >=15`

Reply:
See https://nodejs.org/en/blog/release/v12.13.0/ and https://nodejs.org/en/blog/release/v14.15.0/. Those are versions that were effectively marked as LTS.

Jest uses the same pattern:
https://github.com/facebook/jest/blob/64d5983d20a628d68644a3a4cd0f510dc304805a/package.json#L145-L147

Reply:
I think the `12.* 14.* >=15` is logical `&&`. Not sure though. I have also not found yet references with the `||`.

Reply:
Those versions are when LTS started, it doesn't mean they are getting updates. Looking at the security releases (eg: https://nodejs.org/en/blog/vulnerability/april-2021-security-releases/), only the very last version of `12.*` gets them, so in that sense `12.13.0` is as (in)secure and (un)supported as `12.22.0`. Picking the version when the whole `12.*` branch was moved to LTS sounds arbitrary to me 🤷🏼 

That being said, as we can't' specify "latest 12.*", whatever we add in `engines` packages won't actually prevent them to be installed in unsupported Node versions (even assuming the package managers honors this field by default).

Reply:
I don't think that matters that much since it only prints a warning when the version of Node.js is not satisfied. If you are really concerned about that, we can go with a simple `>=12`.

Reply:
Updated to:

```json
"engines": {
	"node": ">=12"
},
```